### ISSUE TITLE
Custom Build: Command line is empty. Build aborted.

### ISSUE THREAD

**Issue by Lyberta** [eranif/codelite#1539]
I'm trying to generate CodeLite projects from CMake. However, when I hit F7 CodeLite seems to start building but doesn't do anything. If I select "Rebuild" and "Build" several times, eventually I'm getting:

Command line is empty. Build aborted.

In Build Tab.

I'm attaching the project files and Makefile.
[BuildFailed.zip](https://github.com/eranif/codelite/files/752462/BuildFailed.zip)

You can generate them yourself from this repository:
https://gitlab.com/ftz/general

**Comment by eranif** [comment]
You should report this to CMake. We don't support generated projects outside of CodeLite

**Comment by ysblokje** [comment]
Sorry to necropost here, but I looked into it and I'm of the oppinion that there's something funky going on. In this case  the project(s)s are created with NoConfig as buildconfig. For some reason codelite refuses to do actual compilation if the project target is a library. If it's an executable it will compile properly. 
I tried this by simply changing Static Library to Executable and clicking compile in the contextmenu of the project.
